### PR TITLE
v3 test changes

### PR DESCRIPTION
**Pull request by FullstackGJJ**

### CODE REVIEW

**Review by saigiridhar21 — COMMENTED** [review]
Were you yet to add tests for all the end points?

**Comment by FullstackGJJ** [comment]
Work in progress. First updating the tests to run and then add more.

File: tests/test_calculations_api.py [hunk 1]
```diff
@@ -6,31 +6,29 @@
...
+from fds.analyticsapi.engines.model.pub_date_parameters import PubDateParameters
+from fds.analyticsapi.engines.model.calculation_status import CalculationStatus
+from fds.analyticsapi.engines.model.calculation_unit_status import CalculationUnitStatus
+from fds.analyticsapi.engines.model.calculation_status_summary import CalculationStatusSummary
 
 import common_parameters
 from common_functions import CommonFunctions
 
+
+@unittest.skip("Outdated for v3")
```
Comment (FullstackGJJ) [inline tests/test_calculations_api.py:31]: 👏

File: tests/common_parameters.py [hunk 2]
```diff
@@ -19,16 +19,16 @@
 spar_benchmark_russell_p_r1000 = "RUSSELL_P:R.2000"
 spar_benchmark_russell_prefix = "RUSSELL"
 spar_benchmark_russell_return_type = "GTR"
-vault_default_document = "PA3_DOCUMENTS:DEFAULT"
-vault_default_account = "Client:/analytics/data/US_MID_CAP_CORE.ACTM"
-vault_start_date = "FIRST_REPOSITORY"
-vault_end_date = "LAST_REPOSITORY"
+vault_default_document = "Client:/aapi/VAULT_QA_PI_DEFAULT_LOCKED"
```
Comment (saigiridhar21) [inline tests/common_parameters.py:22]: I can't comment on the exact line. But can you please change the environment variables to `ANALYTICS_API_QA_USERNAME_SERIAL` and `ANALYTICS_API_QA_PASSWORD` or something similar?

Comment (FullstackGJJ) [reply tests/common_parameters.py:22]: It's in `common_parameters` 😉

**Pull request merged**